src/neuroglancer/widget/segment_set_widget.css

```diff
@@ -54,3 +60,15 @@
   height: 1.2em;
   vertical-align: bottom;
 }
+
+.segment-set-widget button.segment-button.segSelected {
```

Comment:
These two classes could use better names to differentiate them from another. Or, preferably, see if you can find something that works for both segment selection classes. Maybe even more general: Something that works for highlighting (segment) buttons across Neuroglancer?